lib/bandit/http1/adapter.ex

```diff
@@ -367,12 +373,28 @@ defmodule Bandit.HTTP1.Adapter do
     415 => "Unsupported Media Type",
     416 => "Requested range not satisfiable",
     417 => "Expectation Failed",
+    418 => "I'm a teapot",
```

Comment:
Not in 9110 (It IS cute though)

Reply:
[RFC 2324](https://www.rfc-editor.org/rfc/rfc2324#section-2.3.2)

See many language stdlibs, including [Python](https://github.com/python/cpython/blob/7a253103d4c64fcca4c0939a584c2028d8da6829/Lib/http/__init__.py#L129-L130) (and Go, linked in the PR body).